### ISSUE TITLE
Can you share the Postdam data cropping method?

### ISSUE THREAD

**Issue by lauraset** [GeoX-Lab/G-RSIM#1]
Hello, @GeoX-Lab , the data cropping method you mentioned in the TGRS paper is to crop each image (6000 x 6000pixels) to 576 patches (256 x 256 pixels). Whether the cropped patches are overlapped? Can you share it? Thank you very much.

**Comment by GeoX-Lab** [comment]
HI Lauraset:
Thanks for your attention.
First, I cannot share the cropping code since it has many dependent libs. Actually, the code is very simple.
The cropped patches are overlapped.

**Comment by eric-le-12** [comment]
Hi, may I ask how the overlapped region was decided?

**Comment by GeoX-Lab** [comment]
HI,
We use the feature of local patches by random choice so the overlap happened.
However, we control that the center of a patch will not belong to another patch.

**Comment by lauraset** [comment]
Hi, @GeoX-Lab . Thank you. Due to the limit of GPU memory, I tested your method using a batch size of 8 (64 in your paper) and the initial learning rate of 0.001 for self-training and 0.0001 for fine-tuning and I got the highest validation OA of about 0.69 in Postdam dataset.  I wonder whether you have tested the influence of batch size?
